### PR TITLE
[MM-28202] Common Component - Alert Banner

### PR DESCRIPTION
**Pull request by devinbinnie**
#### Summary
Alert Banner Common Component for use by Cloud Billing
Demo included on the System Console > Billing and Account > Subscriptions screen
Should be fully functional, with the close button dismissing the alert.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28202

### CODE REVIEW

File: components/alert_banner.scss [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        color: rgba(var(--sys-center-channel-color-rgb), 0.56);
+        font-size: 16px;
+
+        &::before {
+            margin: 0;
+        }
+    }
+
+    &:hover {
+        background: rgba(0, 0, 0, 0.08);
```
Comment (nickmisasi) [inline components/alert_banner.scss:81]: Probably want a var here for themes

**Comment by matthewbirtch** [comment]
@devinbinnie looks like the test server failed.

File: components/alert_banner.scss (hunk 1 above)
Comment (devinbinnie) [reply components/alert_banner.scss:81]: It's just a dark overlay, this is actually something I meant to bring with @matthewbirtch actually.
There's no hover state specified for these panels, so I added this 8% darken overlay, let me know if that's okay.

**Comment by devinbinnie** [comment]
Still appears to be issues with the CircleCI build running out of memory...
I'll ping you guys when things are fixed.

**Comment by devinbinnie** [comment]
@matthewbirtch @stevemudie @nickmisasi @nevyangelova SpinWick is up and running now :)

**Comment by matthewbirtch** [comment]
Hey @devinbinnie, great work on this. A few minor tweaks to get it perfect. Let me know if you have any questions

image

**Comment by devinbinnie** [comment]
@matthewbirtch Fixed the issues you've specified, though your spec on the size of the hover state is kinda fuzzy, so I've matched what I think makes sense. Let me know if it needs further tweaking.

**Review by matthewbirtch — APPROVED** [review]
Looks great @devinbinnie. Approving with the following changes:

```
.AlertBanner__closeButton {
    padding: 5px;
    height: 28px;
    width: 28px;
}
```

**Review by nevyangelova — APPROVED** [review]
LGTM :)

**Pull request merged**